exercises/secret-handshake/canonical-data.json

```diff
@@ -1,6 +1,17 @@
 {
   "exercise": "secret-handshake",
   "version": "1.1.0",
+  "comments": [
+    " In a discussion in #794 and #335 it has been decided to only include   ",
+    " numbers between 0 and 31 (00000 to 11111) in the canonical test data.  ",
+    "                                                                        ",
+    " This is to allow for different implementations in different tracks and ",
+    " not restricting implementations to use bitwise or modulo operations.   ",
+    "                                                                        ",
+    " It is up to each track to include numbers > 31 in the concrete test    ",
+    " suites. In this case, 32 (100000) should yield the same result as 0,   ",
+    " 33 (100001) should yield the same result as 1, and so on.              "
```

Comment:
This reads to me a bit like tracks are required to add tests for numbers > 31

"Tracks may include additional tests for numbers > 31, in this case..."